packages/core/src/babelPlugins/babel-plugin-redwood-directory-named-import.ts

```diff
@@ -9,7 +10,16 @@ const getNewPath = (value: string, filename: string) => {
   const newImportPath = [dirname, basename, basename].join('/')
 
   try {
```

Comment:
Since `resolveFile` returns null when the path isn't found, I think we can remove the try/catch blocks.
```suggestion

return resolveFile(
      path.join(path.dirname(filename), newImportPath)
    )
```

Reply:
👍